### DISCUSSION TITLE
Suggestion for additional 1 def / skiller functions of !total and !skill commands

### DISCUSSION THREAD

**Discussion by HeadersOSRS** [runelite/runelite#16438] [Ideas]
Hey, 

So I run a large 1 defence skilling community (Pure Crew / "pures" in game) and a suggestion thats come up a lot in our CC is to be able to use !total/!lvl skill commands and have it pull the data from the new 1 defence / level 3 skiller hiscores instead of the main hiscores

This could work similar to the way the iron man commands work (by default shows rank and data from the specific iron man build hiscores instead). It could also be achieved by adding an extra command like !ptotal (pure total), !stotal (skiller total) or !plvl skill etc. This extra command was actually the preffered way to do it by most people in my community when we discussed suggesting the idea

Apologies if this has been suggested before, I did search and check the first few pages and didnt see any similar suggestions, thanks!

**Comment by Cpt-Picard** [comment]
it should also include some form of nomenclature to identify the account as pure since there is no visual indicator for pure accounts such as: Level X :99 Experience : X Pure Rank: X
